tools/lsp/properties.rs

```diff
@@ -347,8 +347,8 @@ mod tests {
     ) -> Option<Vec<PropertyInformation>> {
         let element = crate::server_loop::element_at_position(
             dc,
-            lsp_types::TextDocumentIdentifier { uri: url.clone() },
-            lsp_types::Position { line, character },
+            &lsp_types::TextDocumentIdentifier { uri: url.clone() },
```

Comment:
Given that `TextDocumentIdentifier` wraps a `Url` by value, it would be even better to replace the use of `TextDocumentIdentifier` here with the use of just `&Url` and avoid the clone?